README.md

```diff
@@ -1,2 +1,131 @@
 # analytics-ios-integrations
-Monorepo storing Segment's analytics iOS integrations
+
+Monorepo storing Segment's analytics iOS integrations. You can find additional documentation for this repo in the `guides` directory.
+
+## Requirements:
+To run monorepo or migrate a destination, the following are required:
+
+Xcode 8+
+Xcode command line tools and default must be set under "Locations" in Xcode preferences. (If multiple versions of Xcode are installed on a machine, an occur may occur, and so correct default version must be correctly selected)
+
+And local dependencies:
+
+1- HomeBrew
+2- Cocopods 1.8
+3- RubyGems Bundler
+4- OCLint
+5- Clang-format
+
+Alternatively, run .check.sh to install any missing local dependencies.
```

Comment:
Can we add a comment this needs to be ran from within the Scripts directory.

Reply:
I'll add a note here to check "Usage" on how to run the scripts.